### DISCUSSION TITLE
Can not upgrade to Pro plan

### DISCUSSION THREAD

**Discussion by davidseroussi** [vercel/community#3280] [Feature Question] [answered]
### Question

Hi, I'm having trouble upgrading my team plan to Pro.
Whenever I click upgrade, I get a "Failed to upgrade team. Please try again." error message. I have tried multiple working credit cards, on mobile and desktop.
![Screenshot_20230718-083547](https://github.com/vercel/community/assets/32030405/2b7d3392-0ffb-4d57-91db-a83dc703d8e6)

**Comment by swarnava** [comment, marked as answer]
Hi,

What is the team name which you are trying to upgrade?  Also could you make sure you are upgrading from here https://vercel.com/voicestars?

**Reply by davidseroussi** [reply]
Hi,
The team name I'm trying to upgrade is indeed voicestars.
I just tried again and it worked, did you change something?
Thank you anyway

**Reply by swarnava** [reply]
Hi,

I guess the link above did the trick. The above bug may occur if you try to upgrade the plan from your Hobby account scope. Glad to hear that everything is working as expected. :)
